.gitignore

```diff
@@ -2,3 +2,5 @@
 node_modules/
 dist/
 .parcel-cache/
+env/
+.env
```

Comment:
- [ ] Please your app is not functioning from my end, the links are not working, and the date is not displaying below the navigation bar as requested in the requirements. I am not able to;  add books, display the list of books, display the contact page. Please see image below and kindly fix that.

![blankPage](https://user-images.githubusercontent.com/65852621/155490460-6d520455-1051-4ea9-a61b-d184ebe0a673.png)

Reply:
Hi @Mihndim2020  

You are right, I was running using the below procedure🧾. **_It will work fine using below_**

1. Install our packages `npm install`
2. Now run our app using `npx parcel index.html`

**Can you try the above method**